### PR TITLE
ci: Create intial job for pkgsync

### PR DESCRIPTION
**Pull request by jcvenegas**
Add a minimal job to allow be modified as needed.

Fixes: #585

Signed-off-by: Jose Carlos Venegas Munoz <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

File: cmd/kata-pkgsync/Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,15 @@
+pipeline {
+  agent {
+    node {
+      label 'ubuntu-lts-latest-vexxhost'
+    }
+
```
Comment (jodh-intel) [inline cmd/kata-pkgsync/Jenkinsfile]: Nit: blank line :)

File: cmd/kata-pkgsync/Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  }
+  stages {
+    stage('setup') {
+      steps {
+        sh 'echo setup'
+      }
+    }
+  }
+}
```
Comment (grahamwhaley) [inline cmd/kata-pkgsync/Jenkinsfile]: heh, diff tells me you are missing a line feed on the end ;-)

File: cmd/kata-pkgsync/Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+pipeline {
```
Comment (grahamwhaley) [inline cmd/kata-pkgsync/Jenkinsfile]: needs a copyright header

**Pull request closed without merging**